### PR TITLE
Messages

### PR DESCRIPTION
**Pull request by dkumar**

### CODE REVIEW

**Comment by lawrence1357** [comment]
This is good

File: app/controllers/studygroups_controller.rb [hunk 1]
```diff
@@ -110,4 +109,7 @@ def add
 
     redirect_to welcome_index_path
   end
+
+  def show_unscheduled
```
Comment (alice-wang) [inline app/controllers/studygroups_controller.rb]: take this out pls, unscheduled show page is handled

File: app/views/studygroups/show_unscheduled.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+hey
```
Comment (alice-wang) [inline app/views/studygroups/show_unscheduled.html.erb]: wtf

File: db/migrate/20141020011108_create_studygroups.rb [hunk 3]
```diff
@@ -19,11 +19,13 @@ def change
...
-      t.string :tags, array: true, default: []
-
       t.boolean :unscheduled, default: false
 
       t.timestamps
     end
+
   end
+
+
```
Comment (alice-wang) [inline db/migrate/20141020011108_create_studygroups.rb]: why xtra spaces

**Comment by alice-wang** [comment]
its aight

File: app/controllers/studygroups_controller.rb (hunk 1 above)
Comment (dkumar) [inline app/controllers/studygroups_controller.rb]: Removed. route also removed.

File: app/views/studygroups/show_unscheduled.html.erb (hunk 2 above)
Comment (dkumar) [inline app/views/studygroups/show_unscheduled.html.erb]: Was a placeholder. Removed.

File: db/migrate/20141020011108_create_studygroups.rb (hunk 3 above)
Comment (dkumar) [inline db/migrate/20141020011108_create_studygroups.rb]: Accident, removed.

**Pull request merged**